extensions/schemas/standard/clause_7_schemas.adoc

```diff
@@ -58,12 +58,13 @@ The following recommendations are intended to simplify parsing a schema and to h
 ^|C |Properties that represent a URI SHOULD be represented as a string with format "uri" or "uri-reference".
 ^|D |Properties that represent a URI template SHOULD be represented as a string with format "uri-template".
 ^|E |Properties that represent a UUID SHOULD be represented as a string with format "uuid".
-^|F |For string properties that are, "minLength", "maxLength", "enum" and/or "pattern" SHOULD be provided, where applicable.
+^|F |For string properties, "minLength", "maxLength", "enum" and/or "pattern" SHOULD be provided, where applicable.
 ^|G |For numeric properties, "multipleOf", "minimum", "exclusiveMinimum", "maximum", "exclusiveMaximum" SHOULD be provided, where applicable.
-^|H |For integer properties that represent enumerated values, "enum" SHOULD be provided.
-^|I |For array properties, the property SHOULD consist of items that are strings or numbers.
-^|J |Required properties SHOULD be included in "required".
-^|K |The JSON Schema keywords SHOULD be constrained to the those mentioned in this recommendation and requirement `/req/{req-class}/properties`.
+^|H |For integer properties that represent enumerated values without providing a description for each, "enum" SHOULD be provided.
+^|I |For integer properties that represent enumerated values while providing a description for each, "oneOf" with a combination of "const" and "title" for each entry SHOULD be provided.
```

Comment:
Using "enum" for all lists of enumerated values is easier to parse and more predictable. "oneOf" is always more problematic as one never knows how complex the different options are. And "oneOf" is often not supported in tooling.

Having descriptions for each enum could also be handled as an orthogonal capability. In ldproxy, for example, we support references to a dictionary/object that maps codes/values to titles (we call them codelists). An example: [schema](https://demo.ldproxy.net/daraa/collections/CulturePnt/schema) and a [codelist](https://demo.ldproxy.net/daraa/codelists/daraa%2frel?f=json).